react/features/invite/functions.ts

```diff
@@ -420,20 +434,32 @@ export function invitePeopleAndChatRooms(
         inviteServiceUrl: string,
         inviteUrl: string,
         jwt: string,
-        inviteItems: Array<Object>
+        inviteItems: Array<Object>,
+        peopleSearchTokenLocation?: string,
+        peopleSearchTokenKey?: string,
 ): Promise<any> {
 
     if (!inviteItems || inviteItems.length === 0) {
         return Promise.resolve();
     }
 
+    // Parse all the query strings of the search directory endpoint
+    const params = new URLSearchParams();
+    params.append('jwt', jwt);
```

Comment:
This adds jwt to the URL params. This reverts behavior that was dropped in https://github.com/jitsi/jitsi-meet/commit/9af0003c639d6ac1b4f2981f44c1e165e88a8a05.